src/common/store.ts

```diff
@@ -86,38 +55,34 @@ const getMaxVersion = (v1: string, v2: string): string => (gt(v1, v2) ? v1 : v2)
 
 const projectVersion = getMaxVersion('0.1.3-mantis-wallet', DATADIR_VERSION)
 
-export function createPersistentStore(): Store<StoreData> {
+export function createPersistentStore<T extends object>(
+  options?: ElectronStore.Options<T>,
+): Store<T> {
   const store = new ElectronStore({
-    defaults: defaultData,
     cwd: config.dataDir,
-    migrations,
     // See https://github.com/sindresorhus/electron-store/issues/123
     // eslint-disable-next-line @typescript-eslint/ban-ts-comment
     // @ts-ignore
     projectVersion,
     deserialize: (text: string) => JSON.parse(text),
     watch: true,
-  })
+    ...options,
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  }) as ElectronStore<any>
```

Comment:
Casting this makes the type same as before. We have our custom types for the store and `ElectronStore<T>` type does not account for deep properties.